<launch>

<node name="offline_map_updater" pkg="remover" type="offline_map_updater" output="screen">
    <rosparam param="/large_scale/submap_size">120.0</rosparam>
</node>
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find remover)/rviz/remover.rviz"/>
<arg name="target_seq" default="00"/>
<rosparam command="load" file="$(find remover)/config/apollo_04.yaml" />

</launch>
